lib/anaconda/form_builder_helpers.rb

```diff
@@ -17,6 +18,8 @@ def anaconda( anaconda_field_name, form_options = {} )
         output += "<div class='not_anaconda_dropzone'>"
       end
 
+      output += "<input data-anaconda-file-id=\"true\" type=\"hidden\" name=\"anaconda_file_id\" id=\"#{upload_id}\">"
```

Comment:
I don't see this additional field being used anywhere, so I'm not sure what adding it to the form is doing for us. See also lines:

 - https://github.com/rtcreativegroup/anaconda/pull/2/files#diff-8e0c67849a01aaf521e5cfbc143f41f1R363
 - https://github.com/rtcreativegroup/anaconda/pull/2/files#diff-8e0c67849a01aaf521e5cfbc143f41f1R378

Reply:
Well, now I see that the linked PR for Lightstock (#214) references it.

So then, I'd ask, what's the point of setting `data-anaconda-file-id=true` and the linked lines in the prior comment setting the value of the `id` attr as the form field `value`?

Reply:
`data-anaconda-file-id=true` was necessary to provide the other two referenced lines a consistent (with the other inputs) way of referencing this new input. Those other referenced lines assigned the value to that input so that the input had a value to send in the actual post request.

Now, with that said, I think I ended up going a different path by pulling the `anaconda-file-id` directly from the input within lightstock, and therefore we likely don't need that file-id to be submitted with the request.

TL;DR I can probably remove it now. Leaving it would just provide the ID on the `data-anaconda-file-id` on submit if it were to be need by the user, but perhaps this is overkill?

Reply:
Absolutely agree, there are many ways to reference it. I was just going to consistency with the code that was already there. But how about I go ahead and remove it since it isn't needed for our change. If we need it in the form submit down the road we can add it back.